vgsshow/src/main/java/com/verygoodsecurity/vgsshow/core/helper/ViewsStore.kt

```diff
@@ -38,6 +40,7 @@ internal class ViewsStore {
             } else {
                 when (view) {
                     is VGSTextView -> view.setText(revealedData)
+                    is VGSPDFView -> view.render(Base64.decode(revealedData, Base64.NO_WRAP))
                     else -> throw IllegalArgumentException("Not implemented yet!")
```

Comment:
@Endorf We can, but does this make sense? This issue only for development purposes, it can't be achieved in real world, I think we should not over-engineer this.